src/web/modules/gateway_accounts/switch_psp/switch_psp.njk

```diff
@@ -3,9 +3,10 @@
 {% from "govuk/components/radios/macro.njk" import govukRadios %}
 {% from "govuk/components/input/macro.njk" import govukInput %}
 {% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
-
 {% extends "layout/layout.njk" %}
 
+{% set isTestData = not account.live %}
```

Comment:
Should this be used to hide statement descriptor ?

Reply:
Really good suggestion 🙇 https://github.com/alphagov/pay-toolbox/pull/1170/commits/fa853f160e7c35d99731158459e114ba2052e56b